src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,78 @@ metadata_model(
     path = "$PKG.LinearCountRegressor"
 )
 
+const DOC_LINEAR_REGRESSION = "TODO"
+"""
+$(MMI.doc_header(LinearRegressor))
+`LinearRegressor` implements the $DOC_LINEAR_REGRESSION.
```

Comment:
How about we say something like this (please spell check):

---

`LinearRegressor` assumes the target is a continuous variable whose marginal distribution is normal with constant variance, and whose expected value is a linear combination of the features (identity link function). Options exist to specify an intercept or offset feature. 

---

Suggestions for the other models:

`LinearBinaryClassifier` is a [generalized linear model](https://en.wikipedia.org/wiki/Generalized_linear_model#Variance_function), specialised to the case of a binary target variable, with a user-specified link function.   Options exist to specify an intercept or offset feature. 

---

`LinearCountRegressor` is a [generalized linear model](https://en.wikipedia.org/wiki/Generalized_linear_model#Variance_function), specialised to the case of a `Count` binary target variable whose marginal distribution is Poisson, with canonical log link function.  Options exist to specify an intercept or offset feature.

Reply:
This is a great idea!

Reply:
should be resolved in latest commit, leaving unresolved for verification